index.html

```diff
@@ -66,20 +67,26 @@ <h2 class="secondary-text">I'm a Software Developer</h2>
                             <li><a href="#"><img src="./assets/images/medium.svg" alt="Medium"></a> </li>
                         </ul>
                     </div>
-                    <img class="background-image" src="./assets/images/Illustration.svg" alt="header-banner">
+                    
                 </div>
-
-    
+                <img class="background-image" src="./assets/images/Illustration.svg" alt="header-banner">
+                <img src="./assets/images/desktop header2.svg" alt="desktop-banner" class="dekstop-banner">
             </article>
+            
         </section>
         <div class="grid-container">
-            <div class="grid-title">
+            <div class="item0">
                 <div class="title-text">
                     <h1>My Recent Works</h1>
```

Comment:
- [ ] Kindly use **h3** instead of **h1** because it optimizes the main heading of your portfolio website and enhances the user experience. The `h1` tag is specifically designed for main headings, providing a clear and prominent representation of the page's primary topic or purpose. By utilizing the *h3* tag, you create a visual hierarchy that helps users quickly identify and determine the relevance of the content to their needs and interests. This improves the overall usability and navigation of your website, ensuring a more engaging and effective experience for visitors.